src/RA_Leaderboard.h

```diff
@@ -51,18 +65,41 @@ class RA_Leaderboard
     virtual void Submit(unsigned int nScore);
 
 private:
-    const ra::LeaderboardID          m_nID;                 //  DB ID for this LB
+    const ra::LeaderboardID m_nID = ra::LeaderboardID(); //  DB ID for this LB
 
-    void*                            m_pLeaderboard;        //  rc_lboard_t
-    std::shared_ptr<unsigned char[]> m_pLeaderboardBuffer;  //  buffer for rc_lboard_t
-    unsigned int                     m_nCurrentValue = 0;
+    void* m_pLeaderboard = nullptr;                        //  rc_lboard_t
+    std::shared_ptr<unsigned char[]> m_pLeaderboardBuffer; //  buffer for rc_lboard_t
+    unsigned int m_nCurrentValue = 0U;
```

Comment:
Here, the formatter actually undid some manual alignments. I'm okay with that, but the comments should be realigned.